View/TwigView.php

```diff
@@ -0,0 +1,160 @@
+<?php declare(strict_types=1);
```

Comment:
I think it would make sense to extend `DefaultView` here, just to avoid repeating all the calculation logic (as well as `initializePagerfanta()`), don't you agree? All views in Pagerfanta itself do that.

Reply:
I looked at that option.  But:

1) The `Pagerfanta\View\DefaultView` class expects to work against `Pagerfanta\View\Template\TemplateInterface` objects, which are clearly not being used here
2) All the useful bits that are duplicated are private anyway, the only thing that is open for inheritance/overwrite/reuse is `getDefaultProximity`

Reply:
Oh, indeed, I missed that.